app/javascript/dashboard/i18n/locale/en/sla.json

```diff
@@ -61,6 +61,19 @@
         "SUCCESS_MESSAGE": "SLA updated successfully",
```

Comment:
Let us remove the edit i18n keys as well.

Reply:
done